src/sass/_variables.scss

```diff
@@ -0,0 +1,9 @@
+// colors
```

Comment:
This folder and file should become unnecessary once we do things in more of the Bootstrap way.